### PR TITLE
Proof-reading a French page about the OK hand

### PR DESCRIPTION
**Pull request by davidbgk**
I'm not sure of the process to review translation-related PRs, feel free to amend this commit or improve my proposal.

### CODE REVIEW

**Comment by tatianamac** [comment]
Thank you for submitting your first pull request with us, @davidbgk! It looks like you did everything correctly from my review.

I have asked French-speaking folks in the community Discord to provide any feedback.

Of course if you have any French-speaking colleagues/friends who would be willing to offer a second review, that would be lovely too! Stay tuned!

File: 11ty/definitions/fr_FR/signe-ok.md
Comment (tatianamac) [inline 11ty/definitions/fr_FR/signe-ok.md]: I believe it should be n'impliquent as the subject of the verb is plural?

**Review by tatianamac — APPROVED** [review]
One request and approved after!

**Comment by tatianamac** [comment]
@davidbgk Thank you for your help! I got French-speaking community members to review and added their note to the review. Once you make this change, you can update the branch and we'll merge in! Merci bien !

File: 11ty/definitions/fr_FR/signe-ok.md
Comment (davidbgk) [inline 11ty/definitions/fr_FR/signe-ok.md]:
```suggestion
Bien que l'on puisse débattre que les origines du symbole et un grand nombre de leurs utilisations n'impliquent pas de connotations racistes, et que les 4chan-ers « l'ont présenté comme un canular », il est indéniable que le symbole a évolué et est devenu un geste vraiment utilisé par les suprémacistes blancs.
```

**Pull request merged**
